source/features/useful-not-found-page.tsx

```diff
@@ -7,6 +7,8 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import getDefaultBranch from '../github-helpers/get-default-branch';
 import {getCleanPathname} from '../github-helpers';
+import * as api from '../github-helpers/api';
+import GitHubURL from '../github-helpers/github-url';
```

Comment:
```suggestion
import features from '.';
import * as api from '../github-helpers/api';
import GitHubURL from '../github-helpers/github-url';
import getDefaultBranch from '../github-helpers/get-default-branch';
import {getCleanPathname} from '../github-helpers';
```

We sort the from like a tree

Reply:
Done